lib/db/leveldb_dbinstance_updateschema.go

```diff
@@ -34,8 +34,8 @@ func (db *Instance) updateSchema() {
 	if prevVersion < 3 {
 		db.updateSchema2to3()
 	}
-	// This update fixes a problem that only exists in dbVersion 3.
-	if prevVersion == 3 {
+	// This update fixes problems existing in versions 3 to 5
+	if prevVersion < 6 && prevVersion > 2 {
 		db.updateSchema3to4()
```

Comment:
The stuff that is version five on master then needs to become version seven? This feels weird. Can we not assume that noone runs master, increase master to six, and let this be five so the sequence makes sense for users going 48 -> 49 -> 50 (released versions)?